sopa/embedding/patches.py

```diff
@@ -171,6 +172,8 @@ def embed_wsi_patches(
     embedding_image.coords["y"] = patch_width * embedding_image.coords["y"]
     embedding_image.coords["x"] = patch_width * embedding_image.coords["x"]
 
-    sdata.add_image(model_name, embedding_image)
+    shape_key = f"sopa_{model_name}"
+
+    sdata.add_image(shape_key, embedding_image)
 
-    log.info(f"Tissue segmentation saved in sdata['{model_name}']")
+    log.info(f"Tissue segmentation saved in sdata['{shape_key}']")
```

Comment:
Just a quick comment since I see you are changing things. This info message should write something like, WSI embeddings saved in ...